packages/documentation/pages/usage/components/inline-edit-2.vue

```diff
@@ -0,0 +1,84 @@
+<template lang="md">
+<ComponentInfo v-bind="{ component }" />
+
+## Basic
+
+<div class="element-example white">
+	<KtInlineEdit2
+		label="Instructions"
+		placeholder="Click to edit"
+		value="my instructions"
+	/>
+</div>
+
+```html
+<KtInlineEdit2
+	label="Instructions"
+	placeholder="Click to edit"
+	value="my instructions"
+/>
+```
+
+## Multiline
+
+<div class="element-example white">
+	<KtInlineEdit2
+		isMultiLine
+		label="Instructions"
+		placeholder="Edit instructions"
+		value="my instructions"
+	/>
+</div>
+
+```html
+<KtInlineEdit2
+	isMultiLine
+	label="Instructions"
+	placeholder="Edit instructions"
+	value="my instructions"
+/>
+```
+
+## Usage
+
+### Attributes
```

Comment:
No need to document this section anymore IMO, as it’s already auto-generated from `component.meta.typeScript.schema` (assuming you fix the meta of course)